subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -201,12 +212,11 @@ static uint8_t big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bi
 				return BT_HCI_ERR_INVALID_PARAM;
 			}
 
-			/* FIXME: PTO is currently limited to BN */
-			if (!IN_RANGE(pto, 0x00, bn /*0x0F*/)) {
+			if (!IN_RANGE(pto, 0x00, 0x0F)) {
```

Comment:
Coverity will complain about `IN_RANGE` on unsigned integers when the first value is 0x00. 

```suggestion
			if (pto > 0x0F)) {
```

Reply:
Fixed.